include/drivers/pwm.h

```diff
@@ -34,7 +34,8 @@ extern "C" {
  * See @a pwm_pin_set_cycles() for argument description
  */
 typedef int (*pwm_pin_set_t)(struct device *dev, u32_t pwm,
-			     u32_t period_cycles, u32_t pulse_cycles);
+			     u32_t period_cycles, u32_t pulse_cycles,
+			     int flags);
```

Comment:
Another comment from the API meeting: flags ought to be unsigned (e.g. `u32_t`). The implementation here uses `int` since that is what is used for the flags in the Zephyr GPIO API (both on master and on topic-gpio).

Should we consider changing this for GPIO as well?

Reply:
According to @mnkp this change is already planned for GPIO. I have changed the suggested API for PWM flags to use `unsigned int`.